### PR TITLE
intel-one-mono: init at v1.0.0

### PR DESCRIPTION
**Pull request by colemickens**
###### Description of changes

Adds `intel-one-mono`.

repo: https://github.com/intel/intel-one-mono
discussion: https://news.ycombinator.com/item?id=35925124

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by drupol — APPROVED** [review]
You beat me to it!
All good here.

File: pkgs/data/fonts/intel-one-mono/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    hash = "sha256-4XE4HS25TlwXzAh41WJS3d7T2U/qO3Hm4zlvMcgEnpU=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    cd ttf
+    cp *.ttf $out/share/fonts/truetype
```
Comment (drupol) [inline pkgs/data/fonts/intel-one-mono/default.nix:21]: How about copying other formats as well ? Especially the `opentype` ?

**Review by drupol — CHANGES_REQUESTED** [review]

**Comment by felschr** [comment]
I just noticed there are 2 PRs for `intel-one-mono`. I already left some comments on the other PR (#231784) which are also applicable here.

@dpc, @colemickens who wants to continue? Do you both want to maintain this package going forward?

**Comment by colemickens** [comment]
After trying it, I feel I can confidently say that once again, I immediately switched back to Iosevka (iosevka-comfy, actually). No need to continue with this one if someone else is interested in maintaining, they can finish it out.

Thanks for the review and note!

**Pull request closed without merging**